docs/buttons.md

```diff
@@ -103,12 +102,10 @@ subscriptions.attachSmartButton(
 ### Parameters:
 
 - `buttonElement` (Required): HTML button element where smartButton is rendered.
-- `options` (Optional): Accepts JSON object with `theme` attribute.\
-                        i.e.: `{theme: 'dark'|'light'}` (Default is `light`) \
-                        `dark` theme renders the button with dark background and\
-                        light/white text.\
-                        `light` theme renders the button with light/white background \
-                        and dark text.
+- `options` (Optional): Configures appearance of button.
+  - `lang`: (Optional) Sets the button SVG and title.  English is default language.  See [Button API](../src/runtime/button-api.js) for `lang` values.
+  - `theme`: (Optional) Button theme can be `light` (default) or `dark`.
+  - `messageTextColor`: (Optional) Overrides theme color for message text. (ex: `'#09f'`)
```

Comment:
We might want to add a sample image at the end with `transparent` and `messageTextColor` set, once the boq implementation is done.

Reply:
Sounds good!